agent/xds/server.go

```diff
@@ -164,6 +165,8 @@ const (
 )
 
 func (s *Server) process(stream ADSStream, reqCh <-chan *envoy.DiscoveryRequest) error {
+	logger := s.Logger.Named(logging.XDS)
```

Comment:
similar comment here about using `With`

Reply:
I'm not sure that would work here since the proxy ID isn't known until the xDS server state machine is running